internal/manager/run.go

```diff
@@ -54,8 +54,14 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 	if err != nil {
 		return fmt.Errorf("get kubeconfig from file %q: %w", c.KubeconfigPath, err)
 	}
+
+	setupLog.Info("starting standalone health check server")
+	healthServer := &healthCheckServer{}
+	healthServer.setHealthzCheck(healthz.Ping)
+	healthServer.Start(c.ProbeAddr, setupLog.WithName("health-check"))
```

Comment:
Shouldn't we have `readyzCheck` set before running the server as well? Now it will return `404` and we'll get crash loops anyway, no?

Reply:
I think we should add a readyzCheck that will use `healthz.Ping` here as well, but only in case the admin API service is specified.

Reply:
failure on `readyzCheck` (readiness probe) will NOT let kubelet kill the pod: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/ . It will make the container be in the "NotReady" status and not able to accept traffics.